src/test-runs/dto/create-test-request.dto.ts

```diff
@@ -1,9 +1,12 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsOptional, IsBase64, IsUUID, IsNumber, IsBoolean } from 'class-validator';
+import { Transform } from 'class-transformer';
+import { IsOptional, IsUUID, IsNumber, IsBoolean, IsString, IsBase64 } from 'class-validator';
```

Comment:
Codacy found an issue: ['IsString' is defined but never used.](https://app.codacy.com/gh/Visual-Regression-Tracker/backend/pullRequest?prid=6099953)